packages/suite/src/views/dashboard/PortfolioCard/PortfolioCard.tsx

```diff
@@ -139,7 +139,7 @@ export const PortfolioCard = memo(() => {
                 ) : undefined
             }
         >
-            <Card paddingType="none">
+            <Card paddingType="none" contentOverflow="visible">
```

Comment:
I guess this is portfolio chart on dashboard. 

Your issue and screenshot is about chart in an account which is not fixed

![Screenshot 2025-03-11 at 15 04 04](https://github.com/user-attachments/assets/bf8cbf40-c895-4ddc-8af9-a0862c81c7d3)